httpserver/types.go

```diff
@@ -24,7 +24,7 @@ const VerificationResultVersion = "0.1.0"
 type VerificationResponse struct {
 	Version         string        `json:"version"`
 	IsSuccess       bool          `json:"isSuccess"`
-	VerifierReports []interface{} `json:"verifierReports,omitempty"`
+	VerifierReports []interface{} `json:"nestedReports,omitempty"`
```

Comment:
If we updates this name, this will break all the existing rego policies users might have? We would also need to update the policies in the `/library`?

Reply:
Good call, rename it back to `verifierReports`